Sources/Instrumentation/URLSession/README.md

```diff
@@ -0,0 +1,26 @@
+# URL Session instrumentation
+
+This package captures the network calls produced by URLSession. Just by initializing the cl
```

Comment:
what is "cl"?

Reply:
it should say `class`, will change

Reply:
Definetely deleted last sentence, it doesn`t add any useful information